### PR TITLE
feat: rename start_embedded_activities to use_embedded_activities

### PR DESCRIPTION
**Pull request by onerandomusername**
## Summary

rename start_embedded_activities to use_embedded_activities per the official documentation.
## Checklist

- [ ] If code changes were made, then they have been tested
    - [x] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `task lint`
    - [x] I have type-checked the code by running `task pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by shiftinv — COMMENTED** [review]
Permission fields haven't been deprecated in the past (e.g. `manage_emojis` still exists), that's not necessary here either.

**Comment by onerandomusername** [comment]
> Permission fields haven't been deprecated in the past (e.g. `manage_emojis` still exists), that's not necessary here either.

The reason I deprecated this one in particular is because
- embedded applications haven't publicly released
- the actual behavior has changed, from controlling starting activities to using activites.

**Comment by shiftinv** [comment]
> * embedded applications haven't publicly released

But this PR is adding `use_embedded_activities`? Seems fine to me in any case, the permission is known and documented.

> * the actual behavior has changed, from controlling starting activities to using activities.

Same thing with `manage_emojis` which allows you to also manage stickers (the reason it's now an alias of `manage_emojis_and_stickers`).

I'd rather not remove permissions without a major version bump. Deprecating and subsequently removing permission fields might have unwanted effects - methods like `Permissions.update` ignore unknown names, so continued usage of the old name wouldn't result in errors and instead silently pass.

**Comment by onerandomusername** [comment]
> > * embedded applications haven't publicly released
> 
> But this PR is adding `use_embedded_activities`? Seems fine to me in any case, the permission is known and documented.
> 
> > * the actual behavior has changed, from controlling starting activities to using activities.
> 
> Same thing with `manage_emojis` which allows you to also manage stickers (the reason it's now an alias of `manage_emojis_and_stickers`).
> 
> I'd rather not remove permissions without a major version bump. Deprecating and subsequently removing permission fields might have unwanted effects - methods like `Permissions.update` ignore unknown names, so continued usage of the old name wouldn't result in errors and instead silently pass.

That's fair. Do you think its fine to leave them deprecated but not planned for removal?

File: disnake/permissions.py [hunk 1]
```diff
@@ -903,11 +905,24 @@ def send_messages_in_threads(self) -> int:
         return 1 << 38
 
     @flag_value
+    def use_embedded_activities(self) -> int:
+        """:class:`bool`: Returns ``True`` if a user can launch activities (applications
+        with the :attr:`embedded <ApplicationFlags.embedded>` flag) in a voice channel.
```
Comment (shiftinv) [inline disnake/permissions.py:910]:
```suggestion
        """:class:`bool`: Returns ``True`` if a user can use activities (applications
        with the :attr:`~ApplicationFlags.embedded` flag) in a voice channel.
```

File: disnake/permissions.py [hunk 2]
```diff
@@ -903,11 +905,24 @@ def send_messages_in_threads(self) -> int:
...
+
+        .. versionadded:: 2.6
+        """
+        return 1 << 39
+
+    @make_permission_alias("use_embedded_activities")
     def start_embedded_activities(self) -> int:
         """:class:`bool`: Returns ``True`` if a user can launch activities (applications
         with the :attr:`embedded <ApplicationFlags.embedded>` flag) in a voice channel.
```
Comment (shiftinv) [inline disnake/permissions.py:919]:
```suggestion
        """:class:`bool`: An alias for :attr:`use_embedded_activities`.
```

**Comment by shiftinv** [comment]
> Do you think its fine to leave them deprecated but not planned for removal?

I guess that's fine, though it wouldn't be consistent with the others. Maybe don't deprecate them in that sense, but instead just point out that they have new names (smth like "this is an old alias of ...")?

**Comment by onerandomusername** [comment]
> I guess that's fine, though it wouldn't be consistent with the others. Maybe don't deprecate them in that sense, but instead just point out that they have new names (smth like "this is an old alias of ...")?

@shiftinv should be done now

**Review by shiftinv — APPROVED** [review]
lgtm, thanks!

**Pull request merged**